src/main/java/eu/europeana/metis/sandbox/controller/DatasetController.java

```diff
@@ -78,10 +70,16 @@ public DatasetIdDto processDataset(
       @ApiParam(value = "name of the dataset", required = true) @PathVariable(value = "name") String datasetName,
       @ApiParam(value = "country of the dataset", required = true, defaultValue = "Netherlands") @RequestParam Country country,
       @ApiParam(value = "language of the dataset", required = true, defaultValue = "nl") @RequestParam Language language,
-      @ApiParam(value = "dataset records in a zip file", required = true) @RequestParam MultipartFile dataset) {
+      @ApiParam(value = "dataset records uploaded in a zip file", required = false) @RequestParam MultipartFile dataset,
+      @ApiParam(value = "dataset records URL to download in a zip file", required = false) @RequestParam String URL) {
```

Comment:
Agreed the required=false is unnecessary and URL name should be changed.
There is no unclear behavior for the end-user, its the same, for the backend programmer it might be more confusing.
I'll break it up in 2 API calls instead of 1.